public/apps/configuration/app-router.tsx

```diff
@@ -88,22 +84,27 @@ const ROUTE_MAP: { [key: string]: RouteItem } = {
   },
 };
 
-const ROUTE_LIST = [
-  ROUTE_MAP.getStarted,
-  ROUTE_MAP[ResourceType.auth],
-  ROUTE_MAP[ResourceType.roles],
-  ROUTE_MAP[ResourceType.users],
-  ROUTE_MAP[ResourceType.serviceAccounts],
-  ROUTE_MAP[ResourceType.permissions],
-  ROUTE_MAP[ResourceType.tenants],
-  ROUTE_MAP[ResourceType.auditLogging],
-  ROUTE_MAP[ResourceType.tenantsConfigureTab],
-];
+const getRouteList = (multitenancyEnabled: boolean) => {
+  return [
+    ROUTE_MAP.getStarted,
+    ROUTE_MAP[ResourceType.auth],
+    ROUTE_MAP[ResourceType.roles],
+    ROUTE_MAP[ResourceType.users],
+    ROUTE_MAP[ResourceType.serviceAccounts],
+    ROUTE_MAP[ResourceType.permissions],
+    ...(multitenancyEnabled ? [ROUTE_MAP[ResourceType.tenants]] : []),
+    ROUTE_MAP[ResourceType.auditLogging],
+    ...(multitenancyEnabled ? [ROUTE_MAP[ResourceType.tenantsConfigureTab]] : []),
+  ];
+};
 
-const allNavPanelUrls = ROUTE_LIST.map((route) => route.href).concat([
-  buildUrl(ResourceType.auditLogging) + SUB_URL_FOR_GENERAL_SETTINGS_EDIT,
-  buildUrl(ResourceType.auditLogging) + SUB_URL_FOR_COMPLIANCE_SETTINGS_EDIT,
-]);
+export const allNavPanelUrls = (multitenancyEnabled: boolean) =>
+  getRouteList(multitenancyEnabled)
```

Comment:
another (slightly better?) approach would be to filter out tenants related paths if multi-tenancy is enabled. See [also](https://dev.to/sidramaqbool/best-practices-for-using-the-filter-method-in-reactjs-3dog)

Reply:
I think it is fine as is - if a need arises in the future we can consider that, but since tenancy is on its way out the door anyways we can just keep it like this and simply remove it when the time comes.